client/devdocs/design/search-collection.jsx

```diff
@@ -61,7 +62,10 @@ export default React.createClass( {
 		searchString = example.type.displayName;
 
 		if ( this.props.component ) {
-			return example.type.displayName.toLowerCase() !== this.props.component.replace( /-([a-z])/g, '$1' );
+			const exampleName = invoke( example, 'type.displayName.toLowerCase' );
+			const componentName = invoke( this, 'props.component.replace', /-([a-z])/g, '$1' );
```

Comment:
I'm just saying the lodash invoke method is pretty cool. I was unfamiliar. 

Sent from my iPhone

> On Jul 22, 2016, at 1:44 PM, Dennis Snell <notifications@github.com> wrote:
> 
> In client/devdocs/design/search-collection.jsx:
> 
> > @@ -61,7 +62,10 @@ export default React.createClass( {
> >  		searchString = example.type.displayName;
> >  
> >  		if ( this.props.component ) {
> > -			return example.type.displayName.toLowerCase() !== this.props.component.replace( /-([a-z])/g, '$1' );
> > +			const exampleName = invoke( example, 'type.displayName.toLowerCase' );
> > +			const componentName = invoke( this, 'props.component.replace', /-([a-z])/g, '$1' );
> these lines were preventing navigating to individual component pages in the devdocs. I snuck in a fix here so I could make sure my stuff was working, but maybe I should create a separate PR just for these lines
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>